capella_rm_bridge/auditing.py

```diff
@@ -452,34 +485,39 @@ def _count_changes(
 
     def _is_reqtype_change(self, change: Change) -> bool:
         if isinstance(change, (Modification, Deletion)):
-            obj = self.model.by_uuid(change.parent)
+            if match := self.ptrn.match(change.parent):
+                class_name = match.group("ClassName")
+            else:
+                raise ValueError(
+                    "Can't match class name in short representation: "
+                    + change.parent
+                )
```

Comment:
The `change.parent` short representation is directly used in the generated report/log and it can't be retrieved after the change happened. I figured that it all needed information is already in the short representation... so there is no need to increase the attributes on the Change dataclasses even more. In a world where ChatGPT supports in writing regex patterns there is almost no pain involved. This is already simple. So how much does "a lot more simplier" mean here?